src/webui/frontend/src/store/data.ts

```diff
@@ -41,3 +41,19 @@ export class Baseline {
     this.description = (bl.docstring || '').replace(/^[^\n]*\n/g, '');
   }
 }
+
+export class Connection {
+  raw: ConnectionPayload;
+  table_name: string;
+  created_on: string;
+  byte_count: number;
+  row_count: number;
+
+  constructor(cti: ConnectionPayload) {
+    this.raw = cti;
+    this.table_name = cti.name;
+    this.created_on = cti.created_on;
+    this.byte_count = cti.bytes;
+    this.row_count = cti.rows;
+  }
+}
```

Comment:
can you set your editor to add newline at end of file? https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

Reply:
Done.